student/src/main/java/tds/student/services/remote/RemotePrintService.java

```diff
@@ -0,0 +1,360 @@
+    String requestParameters = "FileFormat:" + mainBrailleAttachment.getType().toUpperCase(); // name:value;name:value
+
+    boolean isTranscript = isTranscriptRequest(brailleAttachments);
+
+    if (isTranscript) {
+      requestValue += ";" + brailleAttachments.get(1).getFile();
+    }
+
+    String requestDescription = String.format("%1$s (%2$s)", getPassageLabel(pageGroupToPrint, isTranscript),
+      mainBrailleAttachment.getType());
+
+    OpportunityInstance opportunityInstance = testOpp.getOppInstance();
+
+    final ExamPrintRequest request = new ExamPrintRequest.Builder(UUID.randomUUID())
+      .withExamId(opportunityInstance.getExamId())
+      .withSessionId(opportunityInstance.getSessionKey())
+      .withPagePosition(pageGroupToPrint.getNumber())
+      .withItemPosition(PASSAGE_PRINT_ITEM_POSITION_DEFAULT)
+      .withType(requestType)
+      .withDescription(requestDescription)
+      .withParameters(requestParameters)
+      .withValue(requestValue.replace("\\", "\\\\"))
+      .build();
+
+    examRepository.createPrintRequest(request);
+
+    isSuccessful = true;
+    pageGroupToPrint.setPrinted(isSuccessful);
+
+    return isSuccessful;
+  }
+
+  @Override
+  public boolean printItemBraille(TestOpportunity testOpp, ItemResponse responseToPrint, AccLookup accLookup) throws ReturnStatusException {
+    boolean isSuccessful = false;
+
+    if (isLegacyCallsEnabled) {
+      isSuccessful = legacyPrintService.printItemBraille(testOpp, responseToPrint, accLookup);
+    }
+
+    //This isn't ideal, but due to the way the progman properties are loaded within the system this lives within the service rather than the callers.
+    if (!isRemoteExamCallsEnabled) {
+      return isSuccessful;
+    }
+
+    String xmlPath = responseToPrint.getFilePath();
+
+    if (StringUtils.isEmpty(xmlPath)) {
+      throw new ReturnStatusException(String.format("PrintItemBraille: Invalid xml file path for item %1$s.", responseToPrint.getItemID()));
+    }
+
+    IITSDocument document = responseToPrint.getDocument();
+
+    // try and load document if it isn't already loaded
+    if (document == null) {
+      document = contentService.getContent(xmlPath, accLookup);
+
+      if (document == null) {
+        return false;
+      }
+
+      responseToPrint.setDocument(document);
+    }
+
+    ITSContent content = document.getContent(testOpp.getLanguage());
+
+    if (content == null) {
+      return false;
+    }
+
+    // get attachments for the accommodations braille type
+    List<ITSAttachment> brailleAttachments = content.GetBrailleTypeAttachment(accLookup);
+
+    if (brailleAttachments.isEmpty()) { // no need for null check, this is initialized as an empty list
+      LOG.warn("PrintItemBraille: Cannot find a matching braille attachment for the test {} and item {}.",
+        testOpp.getTestKey(), responseToPrint.getItemID());
+      return false;
+    }
+
+    ITSAttachment mainBrailleAttachment = brailleAttachments.get(0);
```

Comment:
Is there a way to extract a common method between this and the block in 207?  It is hard to pick the nuanced difference but this method and print passage braille seem have some overlap.